zh_tw/news/_posts/2024-10-28-redos-rexml-cve-2024-49761.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: news_post
+title: "CVE-2024-49761: ReDoS vulnerability in REXML"
```

Comment:
Ah I just notice this title is not translated, but we can do this in a follow-up PRs on the previous ReDoS posts.

Reply:
Thank you, I will open PRs to translate these news titles.